tests/integration/component.go

```diff
@@ -51,8 +51,8 @@ func componentTests(args ...string) {
 			Expect(output).NotTo(ContainSubstring("Specify output format, supported format: json"))
 		})
 
-		It("Help for odo create should contain machine output", func() {
-			output := helper.CmdShouldPass("odo", "create", "--help")
+		It("Help for odo component list should contain machine output", func() {
```

Comment:
```
Help for odo component list should contain machine output [It]
    /home/travis/gopath/src/github.com/openshift/odo/tests/integration/component.go:54
    Expected
        <string>: List all components in the current application.
        
        Usage:
          odo list [flags]
        
        Examples:
          # List all components in the application
          odo list
        
        Global Flags:
          -v, --v Level              log level for V logs. Level varies from 0 to 9 (default 0).
              --vmodule moduleSpec   comma-separated list of pattern=N settings for file-filtered logging
        
    to contain substring
        <string>: Specify output format, supported format: json
```
component list doesn't contain the string `Specify output format, supported format: json`